developer_manual/app/schema.rst

```diff
@@ -81,72 +55,51 @@ In order to revert the migration you can run
 
    ./occ migrations:execute --down dav 20161130090952
 
-By executing up and down you can explicitly apply the changes and roll them back - as long as down() is properly implemented.
+By executing ``up`` and ``down`` you can explicitly apply the changes and roll them back
+- as long as ``down`` is fully implemented. 
 
-Because all migration steps will be executed upon installation there is no explicit need for unit tests.
+.. NOTE:: 
+   Because all migration steps will be executed upon installation, there is no
+   explicit need for unit tests.
 
 4. Bring the migration live
 
-For the real app upgrade process the app version has to be increased to trigger the migrations.
-This will apply all steps which have not yet been executed.
+For the real application upgrade process the application version has to be
+increased to trigger the migrations. This will apply all migrations which have
+not yet been executed.
 
 How are migrations detected and executed
 ========================================
 
-For each app there will be a table holding the list of migrations which have been executed during past upgrades.
-This list is compared to all migration classes in the app's migration folder :file:`appinfo/migrations` and the
-migrations that are not present in the table will be executed in the order of the class names alphabetically.
+For each application there will be a table holding the list of migrations which
+have been executed during previous upgrades. This list is compared to all
+migration classes in the application's migration folder
+:file:`appinfo/migrations`, and the migrations that are not present in the table
+will be executed in alphabetical order of the class' name.
 
 ===============
 Database Schema
 ===============
 
 .. sectionauthor:: Bernhard Posselt <dev@bernhard-posselt.com>
 
-ownCloud uses a database abstraction layer on top of either PDO, depending on the availability of PDO on the server.
-
-The database schema is inside :file:`appinfo/database.xml` in MDB2's `XML scheme notation <http://www.wiltonhotel.com/_ext/pear/docs/MDB2/docs/xml_schema_documentation.html>`_ where the placeholders \*dbprefix* (\*PREFIX* in your SQL) and \*dbname* can be used for the configured database table prefix and database name.
+ownCloud uses a database abstraction layer on top of either `PDO`_, depending on
+the availability of PDO on the server. The database schema is inside
+:file:`appinfo/database.xml` in MDB2's `XML scheme notation`_ where the
+placeholders \*dbprefix* (\*PREFIX* in your SQL) and \*dbname* can be used for
+the configured database table prefix and database name.
 
 An example database XML file would look like this:
 
-.. code-block:: xml
+.. literalinclude:: ../examples/appinfo/database.xml
+   :language: xml
+   :linenos:
+
+To update the tables used by the application, update database.xml and increase
+the application version number in :file:`appinfo/info.xml` to trigger an update.
```

Comment:
I think this is not true any more. Changes in this file are not used any more except for the initial creation.
Or at least that's why I observed when updating from stable9.1 to master, the code would not go there.

@DeepDiver1975 please confirm